llmnrd.c

```diff
@@ -257,7 +270,11 @@ int main(int argc, char **argv)
 		rm_pid_file = true;
 	}
 
-	log_info("Starting llmnrd on port %u, hostname %s\n", port, hostname);
+	log_info("Starting llmnrd on port %u. Assigned hostname(s):\n", port);
+
+	for(name_i = 0; name_i < name_count; ++name_i)
```

Comment:
Add a space after the for keyword:

```suggestion
	for (name_i = 0; name_i < name_count; ++name_i)
```